### PR TITLE
HOSTED PRIVATE CLOUD : vCloud Director (guides: getting started + faq + first vm creation) en-gb and fr version

### PR DESCRIPTION
**Pull request by pg-ovh**
Redo from previous broken PR : https://github.com/ovh/docs/pull/6254
Let me know if there is anything more I can do @OVH-Team-Guides.
Edition done.

Let me know if you want me to replicate the changes in other section guides.

### CODE REVIEW

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-first-vm-creation/images/vcd-vm-clone.png
Comment (bs-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-first-vm-creation/images/vcd-vm-clone.png:1]: Too large

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.en-gb.md [hunk 1]
```diff
@@ -39,62 +39,121 @@ The different sections available are the following:
...
+- `Security Tags`{.action}
 
-In this section, you can find all the network components of your VCD environments: networks, Edge Gateways, Provider Gateways, IP address ranges, datacentre groups, and security tags.
+![Network Overview](images/vcd_overview_networking.gif){.thumbnail}
 
 ### Content Hub
+In this section, you can manage your catalogs: create, delete or download ISO files, OVA templates, or simply use the preconfigured templates available in the OVHcloud catalog.
 
-![Content Hub Overview](images/vcd-content-overview.png){.thumbnail}
+Content:
```
Comment (bs-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.en-gb.md]: This is from "Libraries", not "Content Hub"

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.en-gb.md [hunk 2]
```diff
@@ -39,62 +39,121 @@ The different sections available are the following:
...
```
Comment (bs-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.en-gb.md:130]: Section "Libraries" is missing

**Review by bs-ovh — CHANGES_REQUESTED** [review]
EN: Add missing parts in vcd-getting-started (FR to check)

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-first-vm-creation/images/vcd-vm-clone.png
Comment (OVH-Team-Guides) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-first-vm-creation/images/vcd-vm-clone.png:1]: Fixed

**Comment by OVH-Team-Guides** [comment]
@pg-ovh , always make sure FR and EN versions are aligned. Please check EN and FR versions are ok before I do the duplications.

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @pg-ovh

**Pull request merged**